submissions/codedentwickler/Week1/Fizzbuzz.playground/Contents.swift

```diff
@@ -1,21 +1,21 @@
-//: Playground - noun: a place where people can play
-
-import Foundation
-
-func fizzbuzz(_ i: Int) -> String {
-    let result = (i % 3, i % 5)
-    switch result {
-        case (0, 0):
-            return "FizzBuzz"
-        case (0, _):
-            return "Fizz"
-        case (_, 0):
-            return "Buzz"
-        default:
-            return String(i)
-    }
-}
-
-for number in 1...100 {
-    print(fizzbuzz(number))
-}
+//: Playground - noun: a place where people can play
+
+import Foundation
+
+func fizzbuzz(_ i: Int) -> String {
+    let result = (i % 3, i % 5)

```

Comment:
Something seems off as this looks like you edited someone else's solution (mine 😄 ). I will decline this PR and you resubmit it. Kindly ensure the changes made only reflects on the files in your submission folder.

Reply:
Then how do I remove your solutions off the folder?